src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/VSTelemetryServiceUtility.cs

```diff
@@ -100,9 +100,11 @@ public static class VSTelemetryServiceUtility
                     projectType = NuGetProjectType.XProjProjectJson;
                 }
 
-                // Get package count.
-                var installedPackages = await nuGetProject.GetInstalledPackagesAsync(CancellationToken.None);
-                var installedPackagesCount = installedPackages.Count();
+                // Get package count - don't attempt to get the project.json package count, because it fails on PCL project creation due to concurrency issue 
+                var installedPackagesCount =
+                    projectType == NuGetProjectType.UwpProjectJson ?
```

Comment:
does UwpProjectJson mean all project.json based projects?

Reply:
Yeah. 
The other category is xproj project.json which isn't possible here. 
We need to clean that code up.